page/about.html

```diff
@@ -0,0 +1,231 @@
+                    Partner 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+        </ul>
+    </section>
+
+    <section class="about-intro-container">
+        <header>
+            <ul class="most-top">
+                <li>
+                    <a href="#" class="icon">
+                        <i class="fab fa-facebook fa-fw"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="icon">
+                        <i class="fab fa-twitter fa-fw"></i>
+                    </a>
+                </li>
+                <li class="icon">English</li>
+                <li>
+                    <a href="#" class="icon">My Page</a>
+                </li>
+                <li>
+                    <a href="#" class="icon">Logout</a>
+                </li>
+            </ul>
+            <span class="mobile-menu mob-side-nav">
+                <img src="../assets/icons/menuIcon.svg" alt="Humburgur Menu">
+            </span>
+    
+        </header>
+
+        <section class="home-intro">
+
+            <div class="page-intro">
+                <h1 id="title-desc">International Technology Talk Summit 2022</h1>
+                <div class="opening-desc">
+                    <p>The Global summit brings together the community of experts, academics adn activists who comprise the Creative Commmons affilate network in a different country every tow this year, Wehope to expand our inivitation list including organizationa ans individiduals.</p>
+                </div>
+            </div>
+            
+        </section>
+
+    </section>
+
+    <section class="contact-us" id="contact-us">
+        <p>Please contact us per Email for any further questions about CC Global Summit</p>
+        <a>bush7840@yahoo.com</a>
+    </section>
+
+    <section class="logo" id="logo">
+        <h1>Creative commons Global Summit 2015 Logo</h1>
+        <p>The logo of CC global summit 2015 was decided through the logo competition from 8.,june to 7,July.</p>
+        <img src="../assets/images/company_image.png" alt="">
+    </section>
+
+    <hr>
+
+    <section class="past-events" id="past-events">
+
+        <div class="program-title">
+            <h2 id="feature-title">See the past CC Global Summit</h2>
+            <div class="break-line">
+                <hr>
+            </div>
+        </div>
+
+        <p>Take a look at the last two CC Gloal Summits which toolk place in Buenos Aires and in Warsaw.</p>
+
+        <ul class="event-list">
+            <li class="event-item">
+                <div class="event-container">
+                    <img src="../assets/images/event_1.jpg" alt="Event One">
+                    <div class="image-text">
+                        <h2 class="event-year">2013</h2>
+                        <p class="event-desc">Ice Addis Global Summit in 2021</p>
+                    </div>
+                </div>
```

Comment:
- [ ] One of the requirements for this project is to make your application identical to that provided by Behance. It would be nice to use the `background-color` for the images in this section. Also, try to make the text that is centered on the images align properly. That way, you would be meeting the requirements for the project. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/153716892-1b76fa2c-f541-48b2-abba-aa06e9eb6156.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/153716911-01626599-ffea-4e15-88cf-503f565c11cd.png)

Reply:
Solved, Thanks for remebering